lib/archethic/bootstrap.ex

```diff
@@ -307,7 +307,13 @@ defmodule Archethic.Bootstrap do
     end
 
     tx =
-      TransactionHandler.create_node_transaction(ip, port, http_port, transport, reward_address)
+      TransactionHandler.create_node_transaction(
```

Comment:
Why do you change the formatting ?
We are all using `mix format` but this was not handled by the standard formatter.
To avoid confusion and longer PR review, please use the standard one provided by Mix.

Reply:
my bad it was a change due to debug lines I put.
anyway I removed it